gto-support-androidx-lifecycle/src/main/java/org/ccci/gto/android/common/androidx/lifecycle/CollectionLiveData.kt

```diff
@@ -4,7 +4,8 @@ import androidx.annotation.RestrictTo
 import androidx.lifecycle.LiveData
 
 sealed class CollectionLiveData<T, C : Collection<T>>(
-    private val collection: ChangeAwareCollection<T>
+    private val collection: ChangeAwareCollection<T>,
+    private val synchronous: Boolean
```

Comment:
This may be a dumb question, but what is the particular benefit of allowing CollectionLiveData to be synchronous? What brought forth the need for the change?